packages/gestalt/src/Video.js

```diff
@@ -224,11 +227,13 @@ export default class Video extends PureComponent<Props, State> {
   };
 
   static defaultProps: {|
+    backgroundColor: 'black' | 'transparent',
```

Comment:
nit: repeating this props isn't great, would be better to define once as its own type

Reply:
DONE